XenAdmin/Diagnostics/Checks/PVGuestsCheck.cs

```diff
@@ -95,9 +107,12 @@ protected override Problem RunHostCheck()
             if (string.IsNullOrEmpty(upgradePlatformVersion))
                 return new PoolHasPVGuestWarningUrl(this, _pool);
 
-            if (Helpers.QuebecOrGreater(upgradePlatformVersion))
+            if (Helpers.QuebecOrGreater(upgradePlatformVersion) && !Helpers.YangtzeOrGreater(upgradePlatformVersion))

```

Comment:
This can also be written:
```
if (Helpers.YangtzeOrGreater(upgradePlatformVersion))
    return new PoolHasPVGuestProblem(this, _pool);
if (Helpers.QuebecOrGreater(upgradePlatformVersion)
   return new PoolHasPVGuestWarningUrl(this, _pool);
```